src/azul/__init__.py

```diff
@@ -860,11 +860,11 @@ def portal_db_object_key(self, catalog_source: str) -> str:
 
     @property
     def lambda_layer_bucket(self) -> str:
-        return self.versioned_bucket
+        return self.s3_bucket
```

Comment:
Add operator checklist items for dev and prod to delete layers from the old location and to announce on Slack that all personal deployment need to be upgraded immediately after this was done for `dev`.